### ISSUE TITLE
calling a localhost URL disconnects from OS server

### ISSUE THREAD

**Issue by donovaly** [OpenSlides/OpenSlides#3105]
Take for example you are in an election and need to jump quickly to another one. This is a usual case because counting the votes needs time therefore several elections are run parallel.

I thought I could easily change between elections by calling their URL. For example I am in an election that has this URL:
http://localhost:8000/assignments/7/
As I wanted to jump to the next one, I changed in the browser URL field the URL to
http://localhost:8000/assignments/8
and pressed return.

result: I get a pink message at the bottom of the browser that the connection to the server is lost. However, the desired election opened as I wanted, only the projector preview window is now collapsed (is was shown before).

So either the connection was not lost or what I did was a mistake (I would describe it then in the manuals) or it is a but that the state of the preview window is lost.

**Comment by donovaly** [comment]
I also tried to open the currently running elections in separate browser tabs. When doing this I get the same problem.

**Comment by FinnStutzenstein** [comment]
The projector preview state is not saved, this is correct, but - i think - not important. But that the red message appears is interesting; I'll try to recreate this. But what you have done should work. This is like a reload but just with another assignment to show.

**Comment by emanuelschuetze** [comment]
I can't reproduce it - tested also with portable version on Windows VM.

**Comment by FinnStutzenstein** [comment]
Hmm, I cannot recreate this. Just entering a new URL or using multiple tabs always works. Also with different apps like motions and users in parallel.
@emanuelschuetze May you can reproduce this with the populated db?

**Comment by donovaly** [comment]
I still can reproduce this bug with the latest snapshot. Look at the bottom of the screen during second 4 and 5 of this video:
[http://ftp.lyx.de/OpenSlides/bug3105.zip](http://ftp.lyx.de/OpenSlides/bug3105.zip)

**Comment by emanuelschuetze** [comment]
Reproduced with firefox (chrome does not show it):
This it not a bug. The page reload caused a websocket disconnect and new connect to server. Firefox is not so fast with reload and shows for a short time on the "old" page the red disconnected message. But this is not dramatic for use. Usually, a user does not need to reload or manually change the url.

=> Won't fix

**Comment by donovaly** [comment]
This bug has a milestone. The milestone can be removed because nothing will be done (WONTFIX or is no real bug).
